velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -123,163 +146,169 @@ class ParquetTpchTest : public testing::Test {
     return splits;
   }
 
-  // Write the DuckDB Lineitem TPC-H table to a Parquet file and return the file
-  // location.
-  std::string writeLineitemTableToParquet() const {
-    constexpr std::string_view tableName("lineitem");
-    constexpr int kRowGroupSize = 10'000;
-    const auto& filePath =
-        fmt::format("{}/{}.parquet", tempDirectory_->path, tableName);
-    // Convert decimal columns to double.
-    const auto& query = fmt::format(
-        "COPY (SELECT l_orderkey as orderkey, l_partkey as partkey, l_suppkey as suppkey, l_linenumber as linenumber, "
-        "l_quantity::DOUBLE as quantity, l_extendedprice::DOUBLE as extendedprice, l_discount::DOUBLE as discount, "
-        "l_tax::DOUBLE as tax, l_returnflag as returnflag, l_linestatus as linestatus, "
-        "l_shipdate as shipdate, l_receiptdate as receiptdate, "
-        "l_shipinstruct as shipinstruct, l_shipmode as shipmode, l_comment as comment "
-        "FROM {}) TO '{}' (FORMAT 'parquet', ROW_GROUP_SIZE {})",
-        tableName,
-        filePath,
-        kRowGroupSize);
-    duckDb_->execute(query);
-    return filePath;
-  }
-
-  RowTypePtr getLineitemColumns(std::vector<std::string> names) const {
-    std::vector<TypePtr> types;
-    for (auto colName : names) {
-      types.push_back(lineitemType_->findChild(colName));
+  std::shared_ptr<const RowType> getRowType(
+      const std::string& table,
+      const std::string& filePath) {
+    dwio::common::ReaderOptions readerOptions;
+    parquet::ParquetReader reader(
+        std::make_unique<dwio::common::FileInputStream>(filePath),
+        readerOptions);
+    auto type = reader.typeWithId();
+    auto names = type->type->as<TypeKind::ROW>().names();
+    std::vector<std::shared_ptr<const Type>> types;
+    for (int i = 0; i < type->size(); i++) {
+      types.push_back(createScalarType(type->childAt(i)->type->kind()));
     }
     return ROW(std::move(names), std::move(types));
   }
 
+  std::shared_ptr<const facebook::velox::RowType> getRowType(
```

Comment:
nit: use RowTypePtr for readability and drop `facebook::velox::` prefix throughout the diff